app/views/shared/_navbar.html.erb

```diff
@@ -16,8 +16,8 @@
       <li class="nav-item dropdown">
         <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
```

Comment:
Can we remove or change Seb's image?

Reply:
Yes good idea!

Reply:
My logo in there for now :) 

<img width="1415" alt="Screenshot 2020-11-26 at 12 33 27" src="https://user-images.githubusercontent.com/71018160/100351741-d5718380-2fe3-11eb-8a27-a75e93f64819.png">